### PR TITLE
{bio}[foss/2018b] SPAdes v3.13.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/5330bcb109916bbdcc9bfe3f533de200 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2436.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f8f9d4b335acf4326647b2f42499c629 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/e01c8e1252c1e43e99799d21d42c55ba for a full test report.

File: easybuild/easyconfigs/s/SPAdes/SPAdes-3.13.0-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+separate_build_dir = True
+
+configopts = ' -DBoost_NO_BOOST_CMAKE=ON'
+
+sanity_check_commands = [('spades.py', '--test')]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['spades-bwa', 'spades-hammer',
+                                     'spades-ionhammer', 'spades.py']],
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/SPAdes/SPAdes-3.13.0-foss-2018b.eb]: there are quite a few more things in `bin`, and at least `spades-core` looks important (?), should this list be updated/extended?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bfd404a8fcff0daadadcee6929c5b034 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/cdaafe23e73f65e852ad14879a89aa4d for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**